Does the next version of the developer pack include everything needed from the previous version?

For instance, does the .NET Framework 4.8 Developer Pack include the needed files from the .NET Framework 4.7.2 Developer Pack and below, such that it allows you to target earlier versions of the .NET Framework?  i.e. the .NET Framework 4.8 Developer Pack supersedes the the .NET Framework 4.7.2 Developer Pack and below.

It does not seem clear to me.
For instance, this `When building an application from Visual Studio or using MSBuild from the command line, MSBuild may display error MSB3644, "The reference assemblies for framework "framework-version" were not found." To address the error, download the developer pack or the targeting pack for that version of .NET Framework.` indicates to me that 4.8 does not include 4.7, etc.
However, the image from the Visual Studio installer appears to indicate that it is not always true as the one highlighted portion states ".NET Framework 4 - 4.6 development tools", and 4.7.2 and 4.8 are not even listed.

I appreciate any clarity you can provide.
Thank you

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf46f86d-940f-3d1b-a83c-6ced0ead20bb
* Version Independent ID: 56305860-d431-de58-1503-cdf57e4814b5
* Content: [Install the .NET Framework developer pack or redistributable - .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/install/guide-for-developers)
* Content Source: [docs/framework/install/guide-for-developers.md](https://github.com/dotnet/docs/blob/main/docs/framework/install/guide-for-developers.md)
* Product: **dotnet-framework**
* Technology: **dotnet-install**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

--- a/docs/framework/install/guide-for-developers.md
+++ b/docs/framework/install/guide-for-developers.md
@@ -46,6 +46,8 @@ Use the following table for quick links, or read further for details. To view th
 
 You can install the **Developer Pack** for a specific version of the .NET Framework, if one is available, on all supported platforms.
 
+**Developer Packs** only target a specific version of .NET Framework and don't include previous versions. For example, the .NET Framework 4.8 Developer Pack wouldn't include .NET Framework 4.7.
+
 You can install the **Web or Offline installer** on:
 
 - Windows 8.1 and earlier

--- a/docs/framework/install/guide-for-developers.md
+++ b/docs/framework/install/guide-for-developers.md
@@ -1,7 +1,7 @@
 ---
 title: "Install the .NET Framework developer pack or redistributable"
 description: "Developers can download and install the .NET Framework developer pack and targeting pack. You can include the .NET Framework redistributable with your apps."
-ms.date: 10/06/2021
+ms.date: 02/23/2022
 helpviewer_keywords:
   - ".NET Framework redistributable package, downloading"
   - ".NET Framework, installing"
@@ -46,7 +46,7 @@ Use the following table for quick links, or read further for details. To view th
 
 You can install the **Developer Pack** for a specific version of the .NET Framework, if one is available, on all supported platforms.
 
-**Developer Packs** only target a specific version of .NET Framework and don't include previous versions. For example, the .NET Framework 4.8 Developer Pack wouldn't include .NET Framework 4.7.
+**Developer Packs** only target a specific version of .NET Framework and don't include previous versions. For example, the .NET Framework 4.8 Developer Pack doesn't include .NET Framework 4.7.
 
 You can install the **Web or Offline installer** on:
 

